package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wmt-probation-rules",
-  "version": "0.1.12",
+  "version": "2.0.0",
```

Comment:
Breaking change because we've moved calculateSdrPoints etc into probation rules?